dev/benchmarks/macrobenchmarks/android/build.gradle

```diff
@@ -13,10 +13,12 @@ allprojects {
     }
 }
 
-rootProject.buildDir = '../build'
+Directory newBuildDir = rootProject.layout.buildDirectory.dir("../../build").get()
```

Comment:
The change to use [`Directory.dir`](https://docs.gradle.org/current/javadoc/org/gradle/api/file/Directory.html) makes this start off as a child directory, so we need one more `../`